Python/Pr_workflow/src/graphqlapi.py

```diff
@@ -0,0 +1,62 @@
+
+class Queries:
```

Comment:
Docstring for Class and the methods are recommended.

Reply:
done